tests/test_files/yaml/multi_llm_streaming_all_pipeline.yml

```diff
@@ -0,0 +1,64 @@
+components:
+  prompt_1:
+    init_parameters:
+      required_variables: "*"
+      template: |
+        "Answer this question: {{query}}
+        Answer:
+        "
+    type: haystack.components.builders.prompt_builder.PromptBuilder
+
+  llm_1:
+    init_parameters:
+      api_base_url: null
+      api_key:
+        env_vars:
+          - OPENAI_API_KEY
+        strict: true
+        type: env_var
+      generation_kwargs: {}
+      model: gpt-4o-mini
+      streaming_callback: null
+      system_prompt: null
+    type: haystack.components.generators.openai.OpenAIGenerator
```

Comment:
Since at some point, we'll remove Generators and use Chat Generators instead, it's better not to introduce new code based on them

Reply:
True, but WDYT about open an issue and change all `OpenAIGenerator` to `OpenAIChatGenerator` in a separate PR?